README.md

```diff
@@ -124,3 +124,11 @@ Docker volumes are created with the prefix `micro-focus-idol-lema_`, which can b
 
 All containers connect to a Docker network called `micro-focus-idol-lema_main`.  The
 `micro-focus-idol-lema` prefix can be changed using the `COMPOSE_PROJECT_NAME` setting.
+
+## Deployment teardown
+
+Stopping and removing of deployed services can be performed by running Python `deploy.py` tool with no arguments:
+
+```
+python3 deploy.py
+```
```

Comment:
would make more sense in the 'Further examples' section, just after the 'To resume a stopped LEMA system [...]' example